examples/persisted-store/src/app/index.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { FC } from 'react';
+import * as ReactDOM from 'react-dom';
+import { openDB } from 'idb';
+import { createClient, fetchExchange, Provider, dedupExchange } from 'urql';
+import { cacheExchange, SerializedEntries } from '@urql/exchange-graphcache';
+import Messages from './Messages';
+
+let db;
+
+const client = createClient({
+  url: 'http://localhost:4000/graphql',
+  exchanges: [
+    dedupExchange,
+    cacheExchange({
+      storage: {
+        read: async () => {
```

Comment:
I think it would be useful if this was separated

```
const myStorageAdapter = {
  read: async () => //...
  write: async () => // ...
};

// ..
exchanges: [dedupExchange, cacheExchange({ storage: myStorageAdapter }), fetchExchange]